Multimedia player: Made volume slider's layout more consistent across browsers.

EricDunsworth opened the pull request:
* Adjusted the volume slider's "thumb" (thick vertical line) dimensions, vertical position, inner colour and background colour to look virtually identical across all mainstream browsers. These changes are especially noticeable in Webkit/Blink browsers.
* Prevented Webkit/Blink browsers from defaulting to using a negative outline offset that overlaps the slider track/thumb.
* Moved volume thumb's width and height SCSS variable declarations into the core variables file.
* Added a border SCSS variable for the volume slider's thumb.

**Screenshots:**

* **Firefox:**
  * **Before:**
![multimedia-controls-firefox-before](https://user-images.githubusercontent.com/1907279/31561314-7b792c3e-b025-11e7-8355-0cae44835c24.png)
  * **After:**
![multimedia-controls-firefox-after](https://user-images.githubusercontent.com/1907279/31561310-776c31c2-b025-11e7-96b9-10e5b9f3f23c.png)
* **Internet Explorer 11:**
  * **Before:**
![multimedia-controls-ie11-before](https://user-images.githubusercontent.com/1907279/31561329-88420ec2-b025-11e7-9934-4cbe6c47116a.png)
  * **After:**
![multimedia-controls-ie11-after](https://user-images.githubusercontent.com/1907279/31561332-8976cabc-b025-11e7-8c19-8f549e35bad1.png)
* **Opera:**
  * **Before:**
![multimedia-controls-opera-before](https://user-images.githubusercontent.com/1907279/31561334-8b0b3fe8-b025-11e7-8d87-3986d531b7ca.png)
  * **After:**
![multimedia-controls-opera-after-v3](https://user-images.githubusercontent.com/1907279/31615533-d98d7c42-b257-11e7-8daa-41c8ff251805.png)

nschonni commented on src/plugins/multimedia/_base.scss:
```diff
@@ -306,13 +303,23 @@ $mm-volume-thumb-width: 10px;
 			width: 7em;
 
+			&:focus {
+				outline-offset: 0; /* Prevents Webkit/Blink browsers from defaulting to a -2px offset. */
```
I'd throw the comment above the line, just for readbility

EricDunsworth committed:
Multimedia player: Made volume slider's layout more consistent across browsers.

* Adjusted the volume slider's "thumb" (thick vertical line) dimensions, vertical position, inner colour and background colour to look virtually identical across all mainstream browsers. These changes are especially noticeable in Webkit/Blink browsers.
* Prevented Webkit/Blink browsers from defaulting to using a negative outline offset that overlaps the slider track/thumb.
* Moved volume thumb's width and height SCSS variable declarations into the core variables file.
* Added a border SCSS variable for the volume slider's thumb.

EricDunsworth commented:
@nschonni Just updated this PR :)!

Specifically:
* Changed "negative outline" to "negative outline offset" in my commit message.
* Moved the comment you mentioned to the line above ``outline-offset: 0;``.
* Revised the aforementioned comment's wording.

EricDunsworth commented:
Just made another minor update.

In the original version of this PR, the thumb in Opera's after screenshot didn't look perfectly vertically-centered relative to its track bar. So I increased the negative offset on Webkit/Blink's thumb and updated the OP's screenshot accordingly.
